### PR TITLE
Adjusted more-actions menu and button to use guidelines.

### PR DESCRIPTION
**Pull request by Ivansss**
It implements what was proposed on #70 .

### CODE REVIEW

File: js/views/roomlistview.js [hunk 1]
```diff
@@ -36,36 +36,61 @@
...
+										'<span class="icon-share svg"></span>'+
+										'<span>'+t('spreedMe', 'Share group')+'</span>'+
+									'</button>'+
+								'</li>'+
+							'</ul>'+
+							'<ul>'+
+								'<li>'+
+									'<button>'+
+										'<span class="icon-close svg"></span>'+
+										'<span>'+t('spreedMe', 'Leave group')+'</span>'+
```
Comment (LukasReschke) [inline js/views/roomlistview.js]: Should be lowercase 'spreedme'

Comment (LukasReschke) [inline js/views/roomlistview.js]: Also at the occurences before

**Comment by Ivansss** [comment]
@LukasReschke changed 😉

**Comment by LukasReschke** [comment]
Thanks! :)

**Pull request merged**